pages/article.js

```diff
@@ -0,0 +1,141 @@
+import Image from 'next/image';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+import { LayoutSingle } from '../components/LayoutSingle';
+import muzeum from '../public/img/museumtrati.jpg';
+import { PortalLink } from '../components/PortalLink';
+import { KeywordLink } from '../components/KeywordLink';
+import Arrow from '../public/img/arrow.svg';
+
+const Article = () => {
```

Comment:
As the article image can get pretty small - 
![CleanShot 2022-04-06 at 10 48 52](https://user-images.githubusercontent.com/12488648/161935582-b92ad0a0-967c-43ce-855d-87b737b303c4.png)

Maybe it would be easy help to open a new tab with the image in full size after a click. Or just a full-size absolute positioned overlay with the image?

Reply:
You mean lightbox?

Reply:
It is small, but portrait image will be ok. You think that I should change the view of "Detaily" on the bigger breakpoint?